roles/common/tasks/python.yml

```diff
@@ -70,6 +70,14 @@
     until: result|succeeded
     retries: 5
 
+  - name: install shade bits for ansible modules
```

Comment:
we should probably create a pthon virtualenv for sensu to use and have everything it needs installed there.